platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/internal/BuildInitDependency.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.buildinit.plugins.internal;
+
+public class BuildInitDependency {
```

Comment:
💅 I would make the class package private and its constructor private

Reply:
Class is also accessed by the maven conversion from package `org.gradleinternal.buildinit.plugins.internal.maven`, constructor changed to private.